src/main/resources/data/tconstruct/scripts/tconstruct/tinker_station.zs

```diff
@@ -0,0 +1,83 @@
+ * 2) Adds a Modifier Recipe that uses 3 Sticks to add the Experienced Modifier to a Pickaxe.
+ * 2.1) The Modifier can be applied 3 times, for a total of Experienced 3.
+ * 3) Adds a Modifier Recipe that uses 3 Apples to add the Sturdy Modifier to any item in the tconstruct:modifiable Item Tag.
+ * 3.1) The Modifier can be applied 5 times, for a total of Sturdy 3.
+ * 3.2) The Recipe requires the Tool that it is being applied on to have the Haste Modifier at level 1 or more.
+ * 4) Adds an Upgrade Modifier Recipe that uses Red Sand to add the Tasty Modifier to any item in the tconstruct:modifiable Item Tag.
+ * 4.1) The Modifier can be applied 3 times, for a total of Tasty 3.
+ * 4.2) The Modifier will use 1 Upgrade slot on the Tool.
+ * 5) Adds an Upgrade Modifier Recipe that uses White Wool to add the Reach Modifier to any item in the tconstruct:modifiable Item Tag.
+ * 5.1) The Modifier can be applied 3 times, for a total of Reach 3.
+ * 5.2) The Modifier will use 2 Upgrade slots on the Tool.
+ * 5.3) The Recipe requires the Tool that it is being applied on to have the Haste Modifier at level 1 or more.
+ * 6) Adds an Ability Modifier Recipe that uses Gravel to add the Smite Modifier to any item in the tconstruct:modifiable Item Tag.
+ * 6.1) The Modifier can be applied 3 times, for a total of Smite 3.
+ * 6.2) The Modifier will use 1 Ability slot on the Tool.
+ * 7) Adds an Ability Modifier Recipe that uses Orange Wool to add the Sharpness Modifier to any item in the tconstruct:modifiable Item Tag.
+ * 7.1) The Modifier can be applied 3 times, for a total of Sharpness 3.
+ * 7.2) The Modifier will use 1 Ability slot on the Tool.
+ * 8) Adds an Incremental Modifier Recipe that uses Black Wool to add the Haste Modifier to any item in the tconstruct:modifiable Item Tag.
+ * 8.1) The Modifier can be applied 3 times, for a total of Haste 3.
+ * 8.2) The Recipe has 8 increments and each Black Wool adds 2 increments, meaning a total of 4 Black Wool will be required per level.
+ * 8.3) If there are any increments left over (if there was somehow 7 increments and another Black Wool is used, for a total of 9 increments), the recipe will return a Black Dye.
+ * 9) Adds an Incremental Modifier Recipe that uses Yellow Wool to add the Sharpness Modifier to any item in the tconstruct:modifiable Item Tag.
+ * 9.1) The Modifier can be applied 3 times, for a total of Sharpness 3.
+ * 9.2) The Recipe has 8 increments and each Yellow Wool adds 3 increments, meaning a total of 3 Yellow Wool will be required per level.
+ * 9.3) If there are any increments left over (if there was 6 increments and another Yellow Wool is used, for a total of 9 increments), the recipe will return a Yellow Dye.
+ * 9.4) The Recipe requires the Tool that it is being applied on to have the Emerald Modifier at level 1 or more.
+ * 10) Adds an Incremental Upgrade Modifier Recipe that uses Blue Wool to add the Bane of Arthropods Modifier to any item in the tconstruct:modifiable Item Tag.
```

Comment:
For incremental recipes, possibly worth mentioning that you should not have two incremental recipes for the same modifier with different sizes. Will lead to unexpected behavior, and the log yells at you. For the example recipes its fine, but for the end user's pack its important to say somewhere